tests/ui/uninlined_format_args.fixed

```diff
@@ -262,3 +262,17 @@ fn tester2() {
         local_i32,
     };
 }
+
+fn literals() {
```

Comment:
I worry about cases where the format string and literal value come from different macro expansions. We should have a test where that is the case.

Reply:
Yeah, i will need tons more of tests for this - some already exist in
* https://github.com/rust-lang/rust-clippy/blob/master/tests/ui/write_literal.rs
* https://github.com/rust-lang/rust-clippy/blob/master/tests/ui/write_literal_2.rs
* https://github.com/rust-lang/rust-clippy/blob/master/tests/ui/write_with_newline.rs

Could you give an example of what specific macro expansion where you thinking about?